news/_posts/releases/https:/build.prestashop.com/news/prestashop-1-7-8-0-rc-release.md

```diff
@@ -0,0 +1,71 @@
+---
+
+layout: post
+title:  "PrestaShop 1.7.8.0 RC1 Is Ready To Be Tested!"
+subtitle: "A few months after the beta, here it is: the first release candidate for PrestaShop 1.7.8.0 is now ready for you to test!"
+date:   2021-07-04
+authors: [ PrestaShop ]
+image: /assets/images/2021/09/178RC-banner_800_418_en.jpg
+icon: icon-leaf
+tags:
+
+* 1.7.8.x
+* version
+* rc
+* minor
+* releases
+* 1.7
+---
+
+![1.7.8 RC is available!](/assets/images/2021/09/178RC-banner_1200_628_en.jpg)
+
+A few months after the beta, the first release candidate for PrestaShop 1.7.8.0 is now ready for you to test!
+
+This is the last step before releasing the final 1.7.8.0. This version aims to verify that all regressions found during the beta have been fixed and that the software is ready for production. If no major or critical regression is reported by October 5, then the final version of 1.7.8.0 will be made available.
+
+Please note that it is not currently possible to perform an automatic upgrade from the Beta to the Release Candidate. And, it will not be possible either to upgrade from this Release Candidate to the final release.
```

Comment:
```suggestion
Please note that it is not currently possible to perform an automatic upgrade from the Beta to the Release Candidate. It will also not be possible to upgrade from this Release Candidate to the final release.
```